pilot/pkg/bootstrap/server.go

```diff
@@ -535,6 +699,29 @@ func (s *Server) initEventHandlers() error {
 	return nil
 }
 
+// Additional istiod functions - inject (new style), galley, certs, ca
```

Comment:
This isn't doing injection like the comment suggest. I suggest we split this up anyways

Reply:
Yes, that was the plan - but iterating. Last commit should have most of it split.